MCprep_addon/materials/generate.py

```diff
@@ -1121,8 +1121,16 @@ def texgen_seus(mat: Material, passes: Dict[str, Image], nodeInputs: List, use_r
 		name="Smooth Inverse",
 		label="Smooth Inverse",
 		location=(-80, -280))
+	sep_node = ""
+	if util.min_bv((5, 0)):
+		sep_node = "ShaderNodeSeparateRGB"
+	elif util.min_bv((3, 3)):
+		sep_node = "ShaderNodeSeparateColor"
+	else:
+		sep_node = "ShaderNodeSeparateRGB"
```

Comment:
Slight nitpick: since 3.3-5.0 is the only exception so far, I think it would make sense to simplify this down:


```suggestion
	sep_node = "ShaderNodeSeparateRGB"
	if util.min_bv((3, 3)) and not util.min_bv((5, 0)):
		sep_node = "ShaderNodeSeparateColor"
```

Reply:
Sure, I also realize that I got it mixed up partially since by tests weren't covering 5.0 like I thought, there's only one conditional now actually, not as crazy as I thought it was initially.